plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/wrapper/LibvirtPatchSystemVmCommandWrapper.java

```diff
@@ -79,6 +79,7 @@ public Answer execute(PatchSystemVmCommand cmd, LibvirtComputingResource serverR
         if (patchResult.first()) {
             String scriptVersion = lines[1];
             if (StringUtils.isNotEmpty(patchResult.second())) {
+                s_logger.debug(String.format("Patch result of systemVM %s: %s", sysVMName, patchResult.second()));
```

Comment:
```suggestion
                logger.debug(String.format("Patch result of systemVM %s: %s", sysVMName, patchResult.second()));
```

Reply:
thanks @JoaoJandre 
confirmed it fixes the build error